src/frontend/src/showcase.ts

```diff
@@ -144,45 +149,34 @@ const iiPages: Record<string, () => void> = {
       cancel: () => console.log("canceled"),
       continue: (alias) => console.log("device alias:", alias),
     }),
-  authenticate: () =>
-    authenticateBoxTemplate({
-      templates: authTemplatesAuthorize({ origin: "https://nowhere.com" }),
-      addDevice: () => console.log("Add device requested"),
-      onContinue: console.log,
-      recoverAnchor: console.log,
-      register: () => console.log("Register requested"),
-      anchors: [BigInt(10000), BigInt(123456)],
-    }),
-  authenticateMany: () =>
-    authenticateBoxTemplate({
-      templates: authTemplatesAuthorize({ origin: "https://nowhere.com" }),
-      addDevice: () => console.log("Add device requested"),
-      onContinue: console.log,
-      recoverAnchor: console.log,
-      register: () => console.log("Register requested"),
-      anchors: [...Array(10).keys()].map((x) => BigInt(10000 + 129 * x * x)),
+
+  // Authorize screens
+
+  authorizeNew: () =>
+    authz.firstTime({ useExisting: () => console.log("Use existing") }),
+  authorizeUseExisting: () => authz.useExisting(),
+  authorizePick: () =>
+    authz.pick({
+      anchors: [BigInt(10000), BigInt(243099)],
+      moreOptions: () => console.log("More options requested"),
     }),
-  authenticateNew: () =>
-    authenticateBoxTemplate({
-      templates: authTemplatesAuthorize({ origin: "https://nowhere.com" }),
-      addDevice: () => console.log("Add device requested"),
-      onContinue: console.log,
-      recoverAnchor: console.log,
-      register: () => console.log("Register requested"),
-      anchors: [],
+  authorizePickAlt: () =>
+    authzAlt.pick({
+      anchors: [BigInt(10000), BigInt(243099)],
+      moreOptions: () => console.log("More options requested"),
     }),
-  authenticateAlternative: () =>
-    authenticateBoxTemplate({
-      templates: authTemplatesAuthorize({
-        origin: "https://nowhere.com",
-        derivationOrigin: "http://jqajs-xiaaa-aaaad-aab5q-cai.ic0.app",
```

Comment:
Could you re-add a screenshot showing alternative origins?

Reply:
There's one, but I forgot to update the screenshot script to click and open the chasm, will do now. Thanks!

Reply:
Thanks! 🙂

Reply:
https://github.com/dfinity/internet-identity/pull/1055/commits/0777f0a5490d0440fbe0611e7a9f4f87374af384